packages/blockchain-tools/src/interfaces/FreighterWallet.ts

```diff
@@ -58,20 +57,26 @@ class FreighterWallet extends ChainBaseClass {
     memo,
   }: { address: string; amount: number; memo: string }) {
     try {
-      console.log("From", this.connectedWallet)
-      console.log("Paying", amount, "XLM to", address, "Memo", memo)
+      console.log("From", this.connectedWallet || '???')
+      console.log("Paying", amount, "XLM to", address, "Memo", memo || '[no]')
       const act = await this.horizon.loadAccount(this.connectedWallet)
+      //const fee = 5000
       const fee = await this.horizon.fetchBaseFee() // 100
-      const opr = StellarSDK.Operation.payment({
+      const xlm = this.fromBaseUnit(amount).toString()
```

Comment:
We discussed the other day that amount should be the "large" unit and we should convert within the `sendPayment` function, did I understand right? In other words, FreighterWallet would be the one wallet that _wouldn't_ convert it.

Reply:
Yes, we should convert in the sendPayment method, I don't know what that conversion does there, will remove it